README.md

```diff
@@ -1,8 +1,17 @@
-![Visual Studio Marketplace Version](https://vsmarketplacebadge.apphb.com/version/Coderoad.coderoad.svg)
-![Visual Studio Marketplace Installs](https://vsmarketplacebadge.apphb.com/installs/Coderoad.coderoad.svg)
-![Visual Studio Marketplace Downloads](https://vsmarketplacebadge.apphb.com/downloads/Coderoad.coderoad.svg)
-
-# CodeRoad VSCode
+<p align="center">
+  <a href="https://coderoad.github.io/">
+    <img alt="CodeRoad Logo" src="https://user-images.githubusercontent.com/9423525/89562564-26b4f780-d7e8-11ea-9eb0-3109b0e55d3b.png" width="100" />
+  </a>
```

Comment:
The formatting on the logo is coming out a bit odd

![image](https://user-images.githubusercontent.com/4660659/89612756-7837a380-d835-11ea-997b-22b10fa45fd0.png)

May be better to put it in front of the title on the left.

Reply:
That's just gh showing changes, here's how it looks w/o the change highlighting:
![image](https://user-images.githubusercontent.com/9423525/89613295-ea1ce680-d84f-11ea-9b5e-be406868de61.png)

Direct link to the file: https://github.com/coderoad/coderoad-vscode/blob/faa88ca683c6e3c7aa8cd4c095ffe47a113a7c44/README.md